include/modules/meta/base.inl

```diff
@@ -249,31 +249,18 @@ namespace modules {
           non_tag = string_util::ltrim(move(non_tag), ' ');
         }
       }
+      m_builder->node(non_tag);
+
+      if (has_tags) {
+        // format-spacing is added between all tags
+        m_builder->spacing(format->spacing);
+      }
+      has_tags = true;
 
       string tag = value.substr(start, end - start + 1);
       bool tag_built = CONST_MOD(Impl).build(&tag_builder, tag);
       string tag_content = tag_builder.flush();
-
-      /*
-       * Remove exactly one space between two tags if the second tag was not built.
-       */
-      if (!tag_built && has_tags && !format->spacing) {
-        if (!non_tag.empty() && non_tag.back() == ' ') {
-          non_tag.erase(non_tag.size() - 1);
-        }
-      }
-
-      m_builder->node(non_tag);
-
-      if (tag_built) {
-        if (has_tags) {
-          // format-spacing is added between all tags
-          m_builder->spacing(format->spacing);
-        }
-
-        m_builder->node(tag_content);
-        has_tags = true;
-      }
```

Comment:
The logic here should not be changed. Any change here that doesn't preserve the exact behavior may add or remove spacing in all of people's modules.